### ISSUE TITLE
Filtering for a string containing a semicolon in the admin panel causes an exception

### ISSUE THREAD

**Issue by alobodig** [keystonejs/keystone#734]
This occurs because the ; character is used to separate filters [here](https://github.com/keystonejs/keystone/blob/66f0bd7/public/js/views/list.js#L279), but that character is not escaped in any way if it occurs in the filter string.

Example exception:
```
non_object_property_load Error thrown for request: /keystone/hits?q=s%3A%24text%3A%3D%3A%3B
TypeError: Cannot read property 'type' of undefined
  at /var/node/mojocms/node_modules/keystone/lib/list.js:812:24
  ...
```

**Comment by JedWatson** [comment]
Should be resolved by the changes we're making in 0.4, will leave this open for confirmation once the filters are working.

**Comment by mxstbr** [comment]
Fixed in 0.4!
